core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/jupyter/CodeGenerationTests.kt

```diff
@@ -52,6 +52,22 @@ class CodeGenerationTests : DataFrameJupyterTest() {
         """.checkCompilation()
     }
 
+    @Test
+    fun `groupBy`() {
+        """
+            val groupBy = dataFrameOf("a")("1", "11", "2", "22").groupBy { expr { "a"<String>().length } named "k" }
+            groupBy.keys.k
```

Comment:
wait, this works in the same cell?

Reply:
No :) The test executes each line individually. It's like 2 separate cells here